packages/mobile/src/notifications/EscrowedPaymentReminderNotification.tsx

```diff
@@ -22,25 +22,25 @@ type Props = OwnProps & WithNamespaces
 
 export class EscrowedPaymentReminderNotification extends React.PureComponent<Props> {
   getCTA = () => {
-    const { payment } = this.props
+    const { payment, t } = this.props
     const recipientPhoneNumber = payment.recipientPhone
     return [
       {
-        text: this.props.t('sendMessage'),
+        text: t('sendMessage'),
         onPress: () => {
           CeloAnalytics.track(CustomEventNames.clicked_escrowed_payment_send_message)
           // TODO: open up whatsapp/text message slider with pre populated message
           try {
             if (Platform.OS === 'android') {
-              SendIntentAndroid.sendSms(recipientPhoneNumber, '')
+              SendIntentAndroid.sendSms(recipientPhoneNumber, t('escrowedPaymentReminderSms'))
             } else {
-              Linking.openURL(`sms:${recipientPhoneNumber}`)
+              Linking.openURL(`sms:${recipientPhoneNumber}&body=${t('escrowedPaymentReminderSms')}`)
```

Comment:
`body` argument used to work, but doesn't anymore.
See https://developer.apple.com/library/archive/featuredarticles/iPhoneURLScheme_Reference/SMSLinks/SMSLinks.html
And https://stackoverflow.com/questions/16558187/iphone-url-scheme-with-content-body-to-sms

I guess just leave a comment about using `MFMessageComposeViewController` instead.

Reply:
Will do!